torch/testing/_internal/opinfo/core.py

```diff
@@ -2030,6 +2029,19 @@ def sample_inputs_elementwise_unary(
                 ),
                 kwargs=op_kwargs,
             )
+    # Create Channels Last Tenosr
+    yield SampleInput(
```

Comment:
Today we have `reference_inputs_elementwise_unary` which contains more interesting tensors, like noncontiguous and arbitrarily strided tensors:

https://github.com/pytorch/pytorch/blob/ac25c210e5452d360fcc8cf5ea96c85756e3e370/torch/testing/_internal/opinfo/core.py#L2282

Should we add channels last tensors to that generator, instead?